src/Configurator/CopyFromPackageConfigurator.php

```diff
@@ -85,20 +111,51 @@ public function copyFile(string $source, string $target)
         if (file_exists($target)) {
             return;
         }
+
         copy($source, $target);
         @chmod($target, fileperms($target) | (fileperms($source) & 0111));
+        $this->write(sprintf(
+            'Created file <fg=green>"%s"</>',
+            $this->relativizePath($target)
+        ));
     }
 
     private function removeFilesFromDir(string $source, string $target)
     {
-        $iterator = new \RecursiveIteratorIterator(new \RecursiveDirectoryIterator($source, \RecursiveDirectoryIterator::SKIP_DOTS), \RecursiveIteratorIterator::CHILD_FIRST);
+        $iterator = new \RecursiveIteratorIterator(
+            new \RecursiveDirectoryIterator($source, \RecursiveDirectoryIterator::SKIP_DOTS),
+            \RecursiveIteratorIterator::CHILD_FIRST
+        );
         foreach ($iterator as $item) {
+            $targetPath = $this->concatenatePathParts([$target, $iterator->getSubPathName()]);
             if ($item->isDir()) {
                 // that removes the dir only if it is empty
-                @rmdir($target.'/'.$iterator->getSubPathName());
+                @rmdir($targetPath);
+                $this->write(sprintf(
+                    'Removed directory <fg=green>"%s"</>',
+                    $this->relativizePath($targetPath)
+                ));
             } else {
-                @unlink($target.'/'.$iterator->getSubPathName());
+                @unlink($targetPath);
+                $this->write(sprintf(
+                    'Removed file <fg=green>"%s"</>',
+                    $this->relativizePath($targetPath)
+                ));
             }
         }
     }
+
+    private function relativizePath(string $absolutePath): string
+    {
+        $relativePath = str_replace(getcwd(), '.', $absolutePath);
+
+        return is_dir($absolutePath) ? rtrim($relativePath, '/').'/' : $relativePath;
```

Comment:
this requires being tested on Windows (which has a different path separator). I'm almost sure your implementation is broken currently.

Reply:
It is a direct copy from [SensioGeneratorBundle](https://github.com/sensiolabs/SensioGeneratorBundle/pull/397/files#diff-0e445b5f16f472d999100a19e18c4e33R102). Did it not work on Windows?